### ISSUE TITLE
[bug #11817] replace SYSCLK with F_CPU

### ISSUE THREAD

**Issue by avrs-admin** [avrdudes/avr-libc#107]
Wed 02 Feb 2005 12:26:12 PM CET

Pls. use only F_CPU in documentation (TWI example).

This issue was migrated from https://savannah.nongnu.org/bugs/?11817

**Comment by avrs-admin** [comment]
Eric Weddington <arcanum>
Mon 07 Feb 2005 10:33:17 PM CET

Please explain:
- What avr-libc version do you see a bug in?
- What the actual bug is?
AFAICT, SYSCLK is used in the documentation and in the TWI example code. Why is F_CPU needed?

**Comment by avrs-admin** [comment]
Joerg Wunsch <joerg_wunsch>
Mon 07 Feb 2005 10:46:57 PM CET

Eric,

I used to use SYSCLK in my private files previously, and
from there, it "diffused" into twidemo.

There seemed to be consensus for F_CPU though, so when I
updated <avr/delay.h> for millisecond/microsecond inline
function, I used that one as the CPU frequency macro.

Now, we should document only one of those, and as the delay
macros constitute an official interface, let's change
twidemo from SYSCLK to F_CPU as well, so people don't get
confused.
